linode/manager
# fix: [M3-7739, M3-7741, M3-7746, M3-7747] - Cherry pick zero price fixes for release

## Description 📝
Eliminates errors displayed in the UI for $0 pricing ($0 is valid) and adds test coverage to flows.

## Changes  🔄
- Linode Migrate dialog displays $0 pricing
- Kubernetes Resize and Add a Node displays $0 pricing and no errors'
- Enable All Backups drawer and enable backups for an individual linode is possible and displays $0 pricing without errors

## How to test 🧪

### Prerequisites
(How to setup test environment)
- Have a linode in `es-madrid`.

### Reproduction steps
(How to reproduce the issue, if applicable)
- In prod, go through the migrate, enable backups, and kubernetes add/resize node pool flows. Note the error messages visible for $0 pricing and in some cases (enable backups on a linode) an inability to complete the action in that region with $0 pricing.

### Verification steps 
(How to verify changes)
- For more thorough testing steps, look at each tagged PR:
  - Test Linode Migrate flow to and from a linode in Madrid #10166 
  - Test Enable All Backups on your account (with a linode in Madrid w\o backups already enabled) and verify total is correct #10161 
  - Test enabling backups on a linode in Madrid #10153 
  - Test a kubernetes cluster in Madrid by adding and resizing a node pool #10157 

## As an Author I have considered 🤔

*Check all that apply*
